src/monitor.js

```diff
@@ -17,7 +17,7 @@ class Monitor {
     this._resourceInterval = null;
   }
 
-  async reportError(err, level='error') {
+  async reportError(err, level='error', tags={}) {
```

Comment:
So let's change the behavior here...

IMO let's make two methods reportError(errOrText, tags) and reportWarning(errOrText, tags)

Note, that if tags for reportError is the string 'warning' we can handle that as a special case for backwards compatibility... and eventually remove support for that...

How does that sound... I know if error and warning levels, if there are more we want to support... maybe it's better to have only reportError(errOrText, level, tags) but then if level is an object set tags= level and level=error..
So you don't have to supply a level argument for error which is the common case...